libnavigation-core/src/test/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetryTest.kt

```diff
@@ -600,6 +638,21 @@ class MapboxNavigationTelemetryTest {
         assertEquals(STOP.type, stopFreeDriveEvent.eventType)
     }
 
+    @Test
+    fun freeDrive_start_and_stop_sent_when_state_changes_from_free_drive_to_active_guidance() {
+        baseMock()
```

Comment:
```suggestion
    fun `freeDrive start+stop sent when state changes from free drive to active guidance`() {
        baseMock()
        every { locationsCollector.lastLocation } returns null
```

The location has to be null to reproduce the issue.

Reply:
check `freeDrive_sent_with_null_location_when_location_not_available_from_free_drive_to_active`

Reply:
:+1: